drink/src/chain_api.rs

```diff
@@ -109,4 +133,25 @@ mod tests {
 
         assert_eq!(sandbox.balance(&DEFAULT_ACTOR), initial_balance);
     }
+
+    #[test]
+    fn runtime_call_works() {
```

Comment:
Can you also include a test where the call fails due to a non-existing pallet?

Reply:
I can't - the method expects a valid `RuntimeCall` object